src/stylesheets/includes/_not-sorted-scss.scss

```diff
@@ -1442,6 +1442,8 @@
 
 .ui.flowing.popup {
   z-index: 10000;
+  margin: 0 !important;
```

Comment:
This will change **all** popups. Are you sure?

Reply:
yea, filter popups are fine